lib/OpenQA/Setup.pm

```diff
@@ -220,6 +224,38 @@ sub update_config {
     }
 }
 
+sub prepare_settings_ui_keys {
+    my ($app) = shift;
+    my $keys = $app->config->{job_settings_ui}->{keys_to_render_as_links};
+    my $settings_ui_cfg;
+    my @link_settings;
+    my @link_keys    = split(/,/, $keys);
+    my $cfgpath      = $ENV{OPENQA_CONFIG} ? path($ENV{OPENQA_CONFIG}) : $app->home->child("etc", "openqa");
+    my $linkkeysfile = $cfgpath->child('settings_ui_links.ini');
+
+    if (-e $linkkeysfile) {
+        $settings_ui_cfg = Config::IniFiles->new(-file => $linkkeysfile->to_string) || undef;
+        $app->config->{link_config} = $settings_ui_cfg;
+
+    }
+    else {
+        $app->log->warn("No configuration file supplied for settings_ui links");
```

Comment:
I would only show such a warning if we would install an empty config file by default. So is this new config file installed by default? I would also state the path where the config file is supposed to be located in the warning.

Reply:
yes, the `settings_ui_links.ini` is commented out, as well as `job_settings_ui` in `openqa.ini`, by default. So i guess is valid. Path added